src/js/angular/clustermanagement/templates/modal/cluster-edit-dialog.html

```diff
@@ -1,130 +1,85 @@
-                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.electionRangeTimeout.$error.pattern">{{'cluster_management.cluster_page.errors.only_positive_integers' | translate}}</div>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="form-group">
-                        <div class="input-group" ng-class="{'has-danger': clusterConfigurationForm.heartbeatInterval.$invalid}">
-                            <label for="heartbeat-interval" class="col-xs-4 col-form-label">{{'cluster_management.cluster_configuration_properties.heartbeat_interval' | translate}}*</label>
-                            <div class="col-xs-8" gdb-tooltip="{{'cluster_management.cluster_configuration_properties.heartbeat_interval_tooltip' | translate}}">
-                                <input class="form-control" placeholder="{{'required.field' | translate}}" type="number" ng-model="clusterConfiguration.heartbeatInterval"
-                                       required id="heartbeat-interval" min="0" pattern="[0-9]+" name="heartbeatInterval">
-                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.heartbeatInterval.$error.required">{{'required.field' | translate}}</div>
-                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.heartbeatInterval.$error.pattern">{{'cluster_management.cluster_page.errors.only_positive_integers' | translate}}</div>
-                            </div>
-                        </div>
-                    </div>
-
-                    <div class="form-group">
-                        <div class="input-group" ng-class="{'has-danger': clusterConfigurationForm.messageSizeKB.$invalid}">
-                            <label for="message-size" class="col-xs-4 col-form-label">{{'cluster_management.cluster_configuration_properties.message_size_kb' | translate}}*</label>
-                            <div class="col-xs-8" gdb-tooltip="{{'cluster_management.cluster_configuration_properties.message_size_kb_tooltip' | translate}}">
-                                <input class="form-control" placeholder="{{'required.field' | translate}}" type="number" ng-model="clusterConfiguration.messageSizeKB"
-                                       required id="message-size" min="0" pattern="[0-9]+" name="messageSizeKB">
-                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.messageSizeKB.$error.required">{{'required.field' | translate}}</div>
-                                <div class="form-control-feedback" ng-show="clusterConfigurationForm.messageSizeKB.$error.pattern">{{'cluster_management.cluster_page.errors.only_positive_integers' | translate}}</div>
+    <div class="modal-body">
+        <form novalidate id="clusterConfigurationForm" name="clusterConfigurationForm">
+            <div class="form-horizontal">
```

Comment:
Here you have a `form-horizontal -> form-group -> div -> form-group`. Is it intentional or it's a mistake?